### DISCUSSION TITLE
Except filter, any else can intercept controller return, just like HandlerInterceptor or ResponseBodyAdvice in Spring?

### DISCUSSION THREAD

**Discussion by lost22git** [micronaut-projects/micronaut-core#4909] [Q&A]
Except filter, any else can intercept controller return, just like HandlerInterceptor or ResponseBodyAdvice in Spring?

**Comment by jameskleeh** [comment]
@zxf-thingsmatrix Your question is not clear to me. You would have to be more specific about what you're asking
